src/components/Nav.astro

```diff
@@ -23,6 +23,15 @@ const currentPath = pathname;
 <script>
   const burger = document.querySelector('button.nav-toggle');
   const navList = document.querySelector('.nav');
+  const navLinks = navList ? navList.querySelectorAll('li') : [];
```

Comment:
You can opt for a logical OR statement if you use optional chaining on the navList object. Up to you, though. Both work just fine but I figured it'd be a little cleaner.